deker/client.py

```diff
@@ -214,16 +214,20 @@ def __init__(
         try:
             set_logging_level(loglevel.upper())
             self.__get_plugins()
-            mem_limit = convert_human_memory_to_bytes(memory_limit)
+            total_available_mem = virtual_memory().total + swap_memory().total
+            memory_limit = convert_human_memory_to_bytes(memory_limit)
+            if memory_limit >= total_available_mem or memory_limit <= 0:
```

Comment:
I find such limitation  unobvious. We do we need to decide for user how much memory he'll be using ? In my opinion, it's better to have it as it was.

Reply:
User is physically limited to the full memory of machine. If he wants 1000 Tb, but has only 100 Gb, how can it become 1000 Tb?